### ISSUE TITLE
Make single command to do all alignment and autofocus

### ISSUE THREAD

**Issue by bgottula** [seeing-things/track#260]
Introduce a new script that automates all of the following steps to prepare for tracking a target:

1. Alignment
2. Guidescope alignment
3. Autofocus

Currently each of these steps are a unique program that must be run separately. Retain the ability to run them separately, but make it possible to run all of them in sequence with a single command.

There are at least a couple of options for how to do this:

- Write a script that just calls `subprocess.run()` to run each program with appropriate command line arguments. This is the most primitive approach, and probably not the cleanest, but might be the quickest to get working.
- Refactor the major programs such that each has a primary function at the module level, and all command-line argument handling is performed outside of that function (probably in `main()`). Then, create new program that calls each of these primary functions one after the other, passing in arguments as needed. I think this is a cleaner approach.
  - This could be a good opportunity to also refactor existing usage of `subprocess.run()`, but that could be deferred to a follow-up issue.

With either approach, I'm not sure exactly how command-line argument handling should be managed. The arguments used at each stage are not necessarily consistent. Perhaps when this program is used, most of the arguments pertaining to the individual steps must be provided in config files and just won't all be exposed at the command line.

- [x] Write the new script
- [x] Test to ensure clean shutdown with CTRL-C or other forms of termination

**Comment by bgottula** [comment]
After some further thought, I think it's better to preserve a separation between setup activities and tracking of a target. Setup only needs to be performed once per session, but a session could include tracking a single target or multiple targets. I updated the description.

**Comment by bgottula** [comment]
I have a script that does the right set of actions and requires minimal user interaction. However it is plagued by my favorite shutdown issues.

When issuing CTRL-C while `align` is running as a subprocess the program ends with the following two log messages emitted:

```
^C[2023-03-25 19:06:12.634] [track] [INFO] Handling SIGINT by raising `SystemExit`
[2023-03-25 19:06:12.634] [track] [INFO] Handling SIGINT by raising `SystemExit`
```

But the two Gemini processes are left running and if a slew is in progress the slew continues without stopping. Clearly this is not working correctly, even with careful use of context managers in the subprocesses. >:( Why does Python make this so incredibly difficult.

**Comment by bgottula** [comment]
Fixed the issue with Gemini processes left running. `subprocess.run()` sends `SIGKILL` to the subprocess if any exception occurs while it's waiting for the subprocess to finish. That's not good. So I replaced `subprocess.run()` with `subprocess.Popen()` and manage it more directly.

While investigating this I also figured out a way to ensure that the Gemini child processes check regularly for parent process aliveness so that even if the parent process dies unexpectedly without performing any cleanup actions the child processes will still bring the mount to a stop and terminate themselves.
